### PR TITLE
Fix code examples

### PR DESCRIPTION
**Pull request by johnhwhite**

### CODE REVIEW

File: projects/docs-tools/src/modules/code-examples/code-examples-editor.service.ts [hunk 1]
```diff
@@ -395,7 +407,7 @@ body {
           moduleResolution: 'node',
           importHelpers: true,
           typeRoots: ['node_modules/@types'],
-          target: 'ES2022',
```
Comment (johnhwhite) [inline projects/docs-tools/src/modules/code-examples/code-examples-editor.service.ts:398]: This change is what ultimately fixed the issue.

File: projects/docs-tools/src/modules/code-examples/code-examples-editor.service.ts [hunk 2]
```diff
@@ -380,6 +391,7 @@ body {
 
     files['tsconfig.json'] = JSON.stringify(
       {
+        compileOnSave: false,
```
Comment (johnhwhite) [inline projects/docs-tools/src/modules/code-examples/code-examples-editor.service.ts:394]: Their Angular demo project sets this.

File: projects/docs-tools/src/modules/code-examples/code-examples-editor.service.ts [hunk 3]
```diff
@@ -354,6 +364,7 @@ body {
     files['package.json'] = JSON.stringify(
       {
         name: 'skyux-spa-demo',
+        version: '0.0.0',
```
Comment (johnhwhite) [inline projects/docs-tools/src/modules/code-examples/code-examples-editor.service.ts:367]: Avoids `undefined` showing during install.

File: projects/docs-tools/src/modules/code-examples/code-examples-editor.service.ts [hunk 4]
```diff
@@ -318,6 +327,7 @@ body {
                 options: {
                   index: 'src/index.html',
                   main: 'src/main.ts',
+                  outputPath: 'dist/demo',
```
Comment (johnhwhite) [inline projects/docs-tools/src/modules/code-examples/code-examples-editor.service.ts:330]: Fixes the project if you download and run it locally.

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**